conf/messages.en

```diff
@@ -2876,3 +2876,37 @@ leaseGrantAmountReceived.change.hidden.individual = How much did you receive for
+ukAndForeignProperty.otherIncomeFromForeignProperty.heading = Other income from foreign property
+ukAndForeignProperty.otherIncomeFromForeignProperty.heading2 = This category only applies to property rental income.
+ukAndForeignProperty.otherIncomeFromForeignProperty.checkYourAnswersLabel = Other income from property
+ukAndForeignProperty.otherIncomeFromForeignProperty.change.hidden = Other income from property
+ukAndForeignProperty.otherIncomeFromForeignProperty.para1.individual = Other income from property can also include:
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet1.individual = rent charges and ground rents
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet2.individual = income from letting others tip waste on your land
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet3.individual = income for the use of a caravan or houseboat at a fixed location
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet4.individual = income from the grant of sporting rights
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet5.individual = receipts from a film crew who pay to film on your land or in your house
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet6.individual = way leaves if the land to which the way leaves relates is used in your property rental business
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet7.individual = local authority grants towards the cost of repairs
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet8.individual = reverse premiums paid to you as an incentive to enter into a lease
+ukAndForeignProperty.otherIncomeFromForeignProperty.para2.individual = How much other income from foreign property did you receive?
+ukAndForeignProperty.otherIncomeFromForeignProperty.para1.agent = Other income from property can also include:
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet1.agent = rent charges and ground rents
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet2.agent = income from letting others tip waste on your land
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet3.agent = income for the use of a caravan or houseboat at a fixed location
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet4.agent = income from the grant of sporting rights
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet5.agent = receipts from a film crew who pay to film on your land or in your house
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet6.agent = way leaves if the land to which the way leaves relates is used in your property rental business
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet7.agent = local authority grants towards the cost of repairs
+ukAndForeignProperty.otherIncomeFromForeignProperty.bullet8.agent = reverse premiums paid to you as an incentive to enter into a lease
+ukAndForeignProperty.otherIncomeFromForeignProperty.para2.agent = How much other income from foreign property did you receive?
```

Comment:
should probably be:
`How much other income from foreign property did your client receive?`